install/install.sh

```diff
@@ -20,18 +20,18 @@
 #   -B, --base-url
 #     Override the base URL used for downloading releases
 
-set -euo pipefail
```

Comment:
Isn't pipefail also nearly universally supported? It might be a bit worse than `local`, I guess.

Reply:
I think `set -o pipefail 2>/dev/null || true` in a separate line should be POSIX compliant.

Reply:
I can add this; it should be okay and will verify. It does fail in dash.

Reply:
done